_grains/keystone_policy.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import salt.config
+import salt.loader
+
+
+def main():
```

Comment:
Because of our mine_functions settings, all of grains items get published to Salt Mine, including this one. We can later use this grain to generate matching policy file on prx nodes as part of OpenStack Dashboard configuration, this feature is WIP as of now.